components/controls/Tooltip.tsx

```diff
@@ -318,3 +403,5 @@ class TooltipPortal extends React.Component {
     return createPortal(this.props.children, this.el);
   }
```

Comment:
Could we move it to its own file ? Would it be wise ? Readability | SRP ...

Reply:
What would we move to its own file?

Reply:
The `TooltipPortal` class

Reply:
I prefer keeping it here. It should _never_ be used on its own. Keeping it part of this file at least leaves it scoped, and prevents ambiguity.